.travis.yml

```diff
@@ -0,0 +1,45 @@
+---
+language: node_js
+os: linux
+dist: bionic # Ubuntu 18.04 LTS
+services:
+  - docker
+
+cache: yarn
+node_js:
+  - lts/* # latest node LTS
+
+before_install:
+  - >
+    docker run
+    -d
+    --name $PGDATABASE-db
+    -p $PGPORT:5432
+    -e POSTGRES_DB=$PGDATABASE
+    -e POSTGRES_USER=$PGUSER
+    -e POSTGRES_PASSWORD=$PGPASSWORD
+    mdillon/postgis:11
+
+  - docker ps -a
+
+script:
+  - mkdir -p "$TEMP_FOLDER" "$ANALYSIS_OUTPUT_DIR"
+  - yarn --ignore-scripts
+  - yarn build:server:prod # compile server "binary"
+  - yarn server:migrate # migrate DB to latest version
+  - yarn test # run test suite (depends on server:migrate)
```

Comment:
should we use test:docker in travis?

Reply:
Travis is already an isolated environment. I think there's a small benefit to running these separately: It might be faster to run due to better caching of yarn stuff than is possible with containers that have to be recreated again and again.
But maybe it's only marginally better?

Reply:
True, let's leave as it is then